### PR TITLE
Updated to Swift 3

### PR DESCRIPTION
**Pull request by andydonzelli**
Hey there. I saw your post on how to make these circular seek bars, but had to change a bunch of stuff to have it work with Swift3. It's mostly minor changes to the syntax, though I did replace your target & selector with a delegate protocol. Let me know what you think.

### CODE REVIEW

File: CircularSeek/ViewController.swift [hunk 1]
```diff
@@ -8,7 +8,12 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+protocol VcDelegate {
```
Comment (karthikkeyan) [inline CircularSeek/ViewController.swift:11]: @andydonzelli You have created a delegate in `ViewController`and exposing that to `CircularSeeker` which makes the `ViewController` and `CircularSeeker` tightly coupled. It supposed to be other way around, `CircularSeeker` have to create a delegate and expose it to the implementer, that way you can make ``CircularSeeker` loosely coupled.

Comment (karthikkeyan) [reply CircularSeek/ViewController.swift:11]: Some thing like this,

```swift
protocol CircularSeekerDelegate: class {
    func circularSeeker(_ seeker: CircularSeeker, didChangeValue value: Float)
}

class CircularSeeker: UIView {
    weak var delegate: CircularSeekerDelegate?

    private func fireValueChangeEvent() {
        delegate?.circularSeeker(self, didChangeValue: currentAngle)
    }
}

```

**Review by karthikkeyan — COMMENTED** [review]
@andydonzelli Thanks for updating this project to Swift 3, I really appreciate that. I have just one comment, once that is addressed then i will merge this PR.

**Comment by andydonzelli** [comment]
Let me know if this is closer to what you had in mind.

**Review by karthikkeyan — APPROVED** [review]

**Pull request merged**